src/test/java/seedu/address/logic/commands/DeleteStallReviewCommandTest.java

```diff
@@ -0,0 +1,45 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandFailure;
+import static seedu.address.testutil.Assert.assertThrows;
+import static seedu.address.testutil.TypicalStalls.getTypicalAddressBook;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.Messages;
+import seedu.address.logic.parser.DeleteStallReviewCommandParser;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+
+/**
+ * Contains integration tests (interaction with the Model) and unit tests for
+ * {@code DeleteStallReviewCommand}.
+ */
+public class DeleteStallReviewCommandTest {
```

Comment:
Good that theres some testing but can add more comprehensive testing next time too such as adding more tests in AddressBookParserTest